blackbox/docs/sql/statements/set-license.rst

```diff
@@ -0,0 +1,60 @@
+.. highlight:: psql
+.. _ref-set-license:
+
+===============
+``SET LICENSE``
+===============
+
+Sets the license Key.
+
+.. rubric:: Table of Contents
+
+.. contents::
+    :local:
+
+Synopsis
+========
+
+::
+
+    SET LICENSE license_key
+
+Description
+===========
+
+``SET LICENSE`` registers a CrateDB license key to the cluster.
```

Comment:
registering  "to the cluster" sounds a bit odd to me.

"registers a license key that will enable CrateDB to operate in Enterprise Edition".